src/ansys/hps/client/jms/resource/task_definition.py

```diff
@@ -54,6 +55,10 @@ class HpcResources(Object):
         Name of the job scheduler queue.
     use_local_scratch : bool, optional
         Whether to use node local storage.
+    native_submit_options : str, optional
+        Additional command line options to pass directly to the scheduler.
+    custom_orchestration_options : dict[str, Union[int, float, str, bool]], optional
+        A dictionary of custom orchestration options.
```

Comment:
```suggestion
        Dictionary of custom orchestration options.
```
We omit the leading article (A, An, The).